tests/test_config.py

```diff
@@ -386,3 +386,56 @@ def test_umask_config(options, expected):
     with AltArgs(cmdline):
         app = NoConfigApp()
     assert app.cfg.umask == expected
+
+
+@pytest.mark.skipif(sys.version_info < (3, 0) or sys.version_info > (3, 6),
```

Comment:
I'm probably missing something obvious, but why we need `sys.version_info > (3, 6)` here?

Reply:
Right - from recollection there were some Py3 versions where this was either missing or the value different.  Either way SSLv3 is very deprecated and we don't really need to test config with it or otherwise encourage its use, so I propose removing this test entirely and have done so.